packages/cursorless-engine/src/actions/actions.types.ts

```diff
@@ -65,4 +65,20 @@ export interface Action {
 /**
  * Keeps a map from action names to objects that implement the given action
  */
-export type ActionRecord = Record<ActionType, Action>;
+export interface ActionRecord extends Record<SimpleActionName, Action> {
+  replaceWithTarget: {
+    run(
+      sources: Target[],
+      destinations: Destination[],
+    ): Promise<ActionReturnValue>;
```

Comment:
Not that we need to solve the world while we're here, but we could prob make these return types stronger now too. But happy to call that out of scope as well

Reply:
I agree. But I think we should do that in the follow up